.circleci/config.yml

```diff
@@ -650,3 +647,6 @@ workflows:
       - e2e-simple:
           requires:
             - build
+      - e2e-mixer:
+          requires:
+            - build
```

Comment:
add the missing new line (and set your editor to not do it again)

Reply:
will do